### ISSUE TITLE
Configuring (sub-)domains end up in a failure

### ISSUE THREAD

**Issue by aerophobic** [stormkit-io/app-stormkit-io#218]
**Describe the bug**
Verification of Domain fails immediately after a new (sub-)domain is set by the user.

**To Reproduce**

1. Go to an environment
2. Click on `configure domain`
3. Set a new (sub-)domain
4. See error (as in the screenshot below)

**Expected behavior**

**Screenshots**

![image](https://user-images.githubusercontent.com/43202397/142981309-8e1b165a-9f56-4637-a18e-23432ed977d9.png)

**Desktop (please complete the following information):**

- Windows 10
- Microsoft Edge
- Version 96.0.1054.29 (Official Build) (64-Bit)

**Additional context**

**Comment by svedova** [comment]
Working on it as I write. This is the improved state: 

![image](https://user-images.githubusercontent.com/3321893/145045497-850e6c02-f52a-47ab-8cbc-5f0e11d5fa93.png)

I added a couple of extra info boxes to help the user. Also, the second step is now only informatory. Previously, we were checking whether the CNAME records point to our servers but for some reason the implementation was not stable. I removed that process.
